src/App.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoListFolder';
 
 import todos from './api/todos';
 import users from './api/users';
 
 function App() {
+  const preparedTodos = [...todos].map(todo => ({
```

Comment:
```suggestion
  const preparedTodos = todos.map(todo => ({
```
map не меняет исходный массив, нет смысла делать копию